tests/filecheck/backend/riscv/canonicalize.mlir

```diff
@@ -157,5 +160,8 @@ builtin.module {
 // CHECK-NEXT:   %add_lhs_rhs_1 = riscv.mul %i1, %add_lhs_rhs : (!riscv.reg<a1>, !riscv.reg<>) -> !riscv.reg<a0>
 // CHECK-NEXT:   "test.op"(%add_lhs_rhs_1) : (!riscv.reg<a0>) -> ()
 
+// CHECK-NEXT:   %and_bitwise_zero = riscv.and %1, 0 : (!riscv.reg<>) -> !riscv.reg<a0>
+// CHECK-NEXT:   "test.op"(%and_bitwise_zero) : (!riscv.reg<a0>) -> ()
+
```

Comment:
I think that you actually want to check that the value passed to `test.op` is coming from the `riscv.mv` that you just created.
Otherwise, if this test pass, this means that the canonicalization pattern actually did not work